mk/lib.mk

```diff
@@ -59,11 +59,11 @@ $(lib-libfile): $(objs)
 	$$(q)rm -f $$@ && $$(AR$(sm)) rcs $$@ $$^
 endif
 ifeq ($(CFG_ULIBS_SHARED),y)
-$(lib-shlibfile): $(objs) $(lib-needed-so-files)
+$(lib-shlibfile): $(objs) | $(lib-needed-so-files)
```

Comment:
Does this really work as intended? As I understand it what's after `|` will be made before this target, but they will not cause this target to be remade if more recent than this target.

Reply:
Ah, right. I misunderstood this order-only thing. I will filter out the `.so` files instead.

Reply:
No worries, I didn't even know about this before. :-)